packages/stores/types.ts

```diff
@@ -83,8 +83,12 @@ type TNotification =
     | ((excluded_until: number) => TNotificationMessage);
 
 type TClientStore = {
+    fetchResidenceList: () => any;
+    fetchStatesList: () => any;
```

Comment:
import { StatesList } from '@deriv/api-types';

```suggestion
    fetchStatesList: () => StatesList;
```

:thinking: